recipes/gocommands/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "gocommands" %}
+{% set version = "0.7.17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cyverse/gocommands/archive/v{{ version }}.tar.gz
+  sha256: 301ab74431a4b5c1559f3c3488965b9053bb7fbccd151d6c499178ea7e3edc44
+
+build:
+  number: 0
+  binary_relocation: False
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+    - conda-forge::ca-certificates
```

```suggestion
    - ca-certificates
```
I don't think this package will help, but you can try it anyways without "conda-forge::". "channel-name::" is not valid syntax for a conda recipe, BTW.